### ISSUE TITLE
another method to implement multiple accounts for the scanner

### ISSUE THREAD

**Issue by bigcross** [AHAAAAAAA/PokemonGo-Map#3048]
## Expected Behavior

We keep the thread number, but set the delay time for each account. So each account will scan in same area but in different time.

## Current Behavior

Due to the Niantic throttling requests, we have to limit -t under 3.

## Possible Solution

We keep each account scan the same large area, but in different time. Set a constant delay time between account loggin. And let each account begin scanning, such as, begin at 0s, 3s, 6s, 9s. And keep them updated the area.

## Steps to Reproduce (for bugs)

1. Set delay time.
2. Loggin with that time
3. Keep -t under 3

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by bigcross** [comment]
The con side for this method is, 
1. you will need a lot of accounts to make sure you won't miss a single pokemon.
2. The first implement can take lot of time if you set your -sq really large.

**Comment by excitedbox** [comment]
I have 100 accounts and I am getting 3 upserts per second using 100 threads. A 20 step limit has just over 1000 steps and takes me about 5:30 to scan plus the login time for all those accounts. It can take around 10 minutes for 1 full cycle. which doesn't leave you much time to get to some pokemon. A staggered scan would leave much less down time because 1 account could log in while another is scanning. Right now there is several minutes of no scanning.

**Comment by bigcross** [comment]
@excitedbox Yes, a staggered scan is what I am thinking about. But there still have some problems, staggered scan required each account sync perfectly, or we have to restart the scanning progress to make sure each area can be take care of.

**Comment by Unvicible** [comment]
There is already a better implementation of multiple accounts underway, It involves a thread per account with independent logins where each account runs at peak optimization.

**Comment by bigcross** [comment]
@Unvicible Yes, I am testing that right now. But I will try to fork one, to test out the performance.
